src/IssueViz.Security/OpenInIDE/Api/StatusResponse.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using Newtonsoft.Json;
+using SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Contract;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Api
+{
+    internal class StatusResponse : IStatusResponse
+    {
+        public StatusResponse(string ideName, string description)
+        {
+            IdeName = ideName ?? throw new ArgumentNullException(nameof(ideName));
+            Description = description ?? throw new ArgumentNullException(nameof(description));
+        }
+
+        [JsonProperty("ideName")]
+        public string IdeName { get; }
+
+        [JsonProperty("description")]
```

Comment:
If we want to change the name of the field, we should have a chat with the team sooner than later